CMakeLists.txt

```diff
@@ -97,8 +97,10 @@ set(LIBMEM_DEPS
 
 if (LIBMEM_BUILD_STATIC)
 	add_library(libmem STATIC ${LIBMEM_SRC})
+	target_compile_definitions(libmem PUBLIC LM_STATIC)
 else()
 	add_library(libmem SHARED ${LIBMEM_SRC})
+	target_compile_definitions(libmem PUBLIC LM_EXPORT)
```

Comment:
Windows users still might would like to rely over LM_EXPORT when building STATIC lib?